src/intervals/functions.jl

```diff
@@ -373,3 +373,13 @@ function nthroot(a::Interval{T}, n::Integer) where T
     b = nthroot(bigequiv(a), n)
     return convert(Interval{T}, b)
 end
+
+"""
+Calculate `x mod y` where `x` is an interval and `y` is a positive divisor.
+"""
+function mod(x::Interval, y::Real)
+    @assert y > zero(y) "modulo is currently implemented only for a positive divisor."
+    division = x / y
+    fl = floor(division)
+    fl.lo < fl.hi ? Interval(zero(y), y) : y * (division - fl)
```

Comment:
Picky suggestions: 
- instead of using `fl.lo` and `fl.hi`, can you use `inf(fl)` and `sup(fl)`? As an alternative, you could test `!isthin(fl)` instead of checking that the bounds are different (`fl.lo < fl.hi`).
- include `return `. 

(These are things we are trying to have thoroughly for version 1.0.)